common/libs/wallet/trezor.ts

```diff
@@ -42,4 +42,27 @@ export default class TrezorWallet extends DeterministicWallet
       );
     });
   }
+
+  public signMessage = () =>
```

Comment:
public signMessage = async () => Error('Signing via Trezor not yet supported.')
Is a little more clean, just nitpicking though.
`new Error` and `Error` do the same thing too.

Reply:
Agree, that's cleaner. Though it causes TypeScript to complain that `signMessage` isn't implementing `IWallet` correctly...